src/main/java/rsb/internal/globval/VarbitIndices.java

```diff
@@ -0,0 +1,15 @@
+package rsb.internal.globval;
+
+public class VarbitIndices {
+    public static final int SPELL_TO_AUTOCAST = 276; // Permitted values: WidgetIndices.DynamicComponents.AutocastSpells
+    public static final int EQUIPPED_WEAPON_TYPE = 357;
+    public static final int DEFENSIVE_AUTOCAST = 2668;
+    public static final int QUICK_PRAYER = 4103;
+    public static final int CURRENT_BANK_TAB = 4150;
+    public static final int XP_DROPS = 4702;
+    public static final int FAIRY_RING_DIAL_ADCB = 3985; // Left dial
```

Comment:
Could this have been 
FAIRY_RING_LEFT_DIAL_ADCB
it'd still be shorter than
FAIRY_RING_LAST_DESTINATION
plus no comments

Reply:
agreed, typo fixed